lua/lazyvim/config/keymaps.lua

```diff
@@ -120,7 +120,7 @@ map("n", "<leader>ud", function() Util.toggle.diagnostics() end, { desc = "Toggl
 local conceallevel = vim.o.conceallevel > 0 and vim.o.conceallevel or 3
 map("n", "<leader>uc", function() Util.toggle("conceallevel", false, {0, conceallevel}) end, { desc = "Toggle Conceal" })
 if vim.lsp.inlay_hint then
```

Comment:
Should we maybe change this to `if vim.lsp.buf.inlay_hint or vim.lsp.inlay_hint`, so that older nightly versions benefit from the keymap as well? As far as I understand, the keymap will only be available if `vim.lsp.inlay_hint` is `true` right now. It doesn't take into account older nightly Neovim versions or am I wrong? Except if Folke decides to drop `vim.lsp.buf.inlay_hint` support all together.

Reply:
I _think_ that should be fine. I'm not sure if @folke had a reason for only checking against `vim.lsp` here and not also including `vim.lsp.buf`.